webapp/src/domain/tests/reducers.ts

```diff
@@ -129,6 +136,10 @@ export const reducer = (state = new TestsState(), action: TestAction) => {
                state.byId = state.byId?.set(action.testId, { ...test, defaultView: action.view })
             }
         }
+        break;
+        case actionTypes.UPDATE_HOOK: {
+            //TODO: define state changes
```

Comment:
This TODO comment might be out of date. I thought the whole premise of redux is to enable easy tracking of state (which involves caching on client side), otherwise you need to manage gui state CRUD operations manually